.travis.yml

```diff
@@ -5,3 +5,5 @@ rust:
   - nightly
 script:
   - /bin/sh run-tests.sh
+  - rustup component add clippy
```

Comment:
I wonder if this could land separately?

Reply:
Yes, I forgot to add cargo clippy --all-features anyway.

Reply:
We probably want both, a run with `--no-default-features` and a run with `--all-features`?